.svglintrc.js

```diff
@@ -163,6 +164,136 @@ module.exports = {
+                    const [x2Coord, y2Coord, xCoord, yCoord] = rest;
+                    // Absolute shorthand curve (S) having the same coordinate as the previous segment and a control point equal to the ending point
+                    if (upperShorthandCurveCommand === command && x1Coord === xPrevCoord && y1Coord === yPrevCoord && x1Coord === x2Coord && y1Coord === y2Coord) {
+                      return true;
+                    }
+                    // Absolute bézier curve (C) having the same coordinate as the previous segment and last control point equal to the ending point
+                    if (upperCurveCommand === command && x1Coord === xPrevCoord && y1Coord === yPrevCoord && x2Coord === xCoord && y2Coord === yCoord) {
+                      return true;
+                    }
+                  }
+
+                  return (
+                    // Absolute horizontal direction (H) having the same x coordinate as the previous segment
+                    (upperHorDirectionCommand === command && x1Coord === xPrevCoord) ||
+                    // Absolute vertical direction (V) having the same y coordinate as the previous segment
+                    (upperVerDirectionCommand === command && x1Coord === yPrevCoord) ||
+                    // Absolute movement (M or L) having the same coordinate as the previous segment
+                    (upperMovementCommands.includes(command) && x1Coord === xPrevCoord && y1Coord === yPrevCoord)
+                  );
+                }
+              }
+            };
+            const invalidSegments = segments.filter(getInvalidSegments);
+
+            if (invalidSegments.length) {
+              invalidSegments.forEach(([command, x1Coord, y1Coord, ...rest]) => {
+                let readableSegment = `${command}${x1Coord}`;
+                if (y1Coord !== undefined) {
+                  readableSegment += ` ${y1Coord}`;
+                }
+                if (curveCommands.includes(command)) {
+                  const [x2Coord, y2Coord, xCoord, yCoord] = rest;
+                  readableSegment += `, ${x2Coord} ${y2Coord}`;
+                  if (yCoord !== undefined) {
+                    readableSegment += `, ${xCoord} ${yCoord}`;
+                  }
+                  if (command === lowerShorthandCurveCommand && (x2Coord !== 0 || y2Coord !== 0)) {
+                    readableSegment += ` (should be "l${x2Coord} ${y2Coord}" or removed)`;
+                  }
+                  if (command === upperShorthandCurveCommand) {
+                    readableSegment += ` (should be "L${x2Coord} ${y2Coord}" or removed)`;
+                  }
+                  if (command === lowerCurveCommand && (xCoord !== 0 || yCoord !== 0)) {
+                    readableSegment += ` (should be "l${xCoord} ${yCoord}" or removed)`;
+                  }
+                  if (command === upperCurveCommand) {
+                    readableSegment += ` (should be "L${xCoord} ${yCoord}" or removed)`;
```

Comment:
Another improvement for resolution tips. What about replace `0.xx` coordinate numbers by their simplified form `.xx`? So, as we are telling the user how to improve the path, improve it by making it efficient.

I've created next function at the start of the file that does the job:

```javascript
function removeLeadingZeroes(number) {
  // convert 0.03 to .03
  return number.toString().replace(/^(-?)(0)(\.?.+)/, '$1$3');
}
```

```suggestion
                    readableSegment += ` (should be "l${removeLeadingZeroes(x2Coord)} ${removeLeadingZeroes(y2Coord)}" or removed)`;
                  }
                  if (command === upperShorthandCurveCommand) {
                    readableSegment += ` (should be "L${removeLeadingZeroes(x2Coord)} ${removeLeadingZeroes(y2Coord)}" or removed)`;
                  }
                  if (command === lowerCurveCommand && (xCoord !== 0 || yCoord !== 0)) {
                    readableSegment += ` (should be "l${removeLeadingZeroes(xCoord)} ${removeLeadingZeroes(yCoord)}" or removed)`;
                  }
                  if (command === upperCurveCommand) {
                    readableSegment += ` (should be "L${removeLeadingZeroes(xCoord)} ${removeLeadingZeroes(yCoord)}" or removed)`;
```

So the message `Unexpected segment "c0 0, 0 0, 0.007 -0.007" in path (should be "l0.007 -0.007" or removed).` (see previous suggestion) would be `Unexpected segment "c0 0, 0 0, 0.007 -0.007" in path (should be "l.007 -.007" or removed).`.